spec/02-integration/04-admin_api/04-plugins_routes_spec.lua

```diff
@@ -450,11 +451,12 @@ for _, strategy in helpers.each_strategy() do
             path = "/schemas/plugins/request-transformer",
           })
           local body = assert.res_status(200, res)
-          assert.match('{"fields":[{', body, 1, true)
+          assert.not_match('"entity_checks":{', body, 1, true)
```

Comment:
I'm not sure how to read this change.
Should the code not assert that the 'fields' JSON key is present and 'entity_checks' is not present?

Reply:
This is a follow-up based on [this assumption](https://github.com/Kong/kong/pull/11108/files#r1244201136). 

When we always include `entity_check` regardless of its contents, this test is verifying if a key named `entity_checks`, is present (analogous to the other assertions)